tsconfig.json

```diff
@@ -1,16 +1,16 @@
 {
+  "extends": "./packages/tsconfig/tsconfig.json",
```

Comment:
This extends should be from `@scaleway/tsconfig` right?

Reply:
Outside of this repo yes :)